lgc/state/LgcContext.cpp

```diff
@@ -234,6 +236,8 @@ LgcContext *LgcContext::Create(LLVMContext &context, StringRef gpuName, unsigned
     return nullptr;
   }
 
+  builderContext->m_targetInfo->getGpuProperty().pageMigrationEnabled = pageMigrationEnabled;
```

Comment:
1. If `pageMigrationEnabled `depends on GFX IP, we have to set it to true or false according to GFX IP value. And this is done by `setTargetInfoXXX`.
2. If `pageMigrationEnabled` depends on driver settings, we have to add a pipeline option in vkgcDefs and in LGC pipeline options. And this is passed from driver to LLPC front-end and finally to LGC.
Anyway, we always avoid adding a new parameter in creation function.

Reply:
Thanks, your suggestion is more reasonable.
It is fixed in the latest commit, please help to review it, thanks !